pytorch_lightning/callbacks/progress.py

```diff
@@ -219,7 +219,7 @@ def on_test_start(self, trainer, pl_module):
     def on_test_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
         self._test_batch_idx += 1
 
-    def on_predict_start(self, trainer, pl_module):
+    def on_predict_epoch_start(self, trainer, pl_module):
```

Comment:
how about back-compatibility?

Reply:
Predict is in beta (not even that, we beta-release it with 1.3)

So we can break it

Reply:
I didn't even realize it was beta

Reply:
Added it back :)